src/Authentication/Controllers/SystemUserController.cs

```diff
@@ -37,7 +37,7 @@ public SystemUserController(ISystemUserService systemUserService)
         /// Returns the list of SystemUsers this PartyID has registered
         /// </summary>
         /// <returns></returns>
-        [Authorize]
+        //[Authorize]
```

Comment:
commented for a reason?

Reply:
nope, just for local test, will fix in new commit